Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "actix-web-opentelemetry"
-version = "0.13.0"
+version = "0.14.0-alpha.1"
```

We can bump the version in the release PR

```suggestion
version = "0.13.0"
```